<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VardiyaSorumluKayit</class>
 <widget class="QWidget" name="VardiyaSorumluKayit">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>740</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{

background: #354152;

}

QLineEdit{
	border: 1px solid gray;
    border-radius: 10px;
    min-width: 6em;
	font-size:20px;
	padding-left:15px;
	height:42px;
	border-radius:15px;
	border-color: #303030;
	background: transparent;
	color:white;


}



QComboBox {
    border: 1px solid gray;
    border-radius: 10px;
    min-width: 6em;
	font-size:20px;
	padding-left:15px;
	height:42px;
	border-radius:15px;
	border-color: #303030;
	background: transparent;
	color:white;

}

QComboBox:on {
    border-bottom-left-radius: 0px;
    border-bottom-right-radius: 0px;
}

QComboBox QAbstractItemView {
    border-bottom-right-radius: 10px;
    border-bottom-left-radius: 10px;
    background: #354152;
    border: 1px solid gray;
	color:white;

}

QComboBox::drop-down {
    border-color: white;
}
QPushButton{
border-radius: 15px;
background-color: qlineargradient(spread:reflect, x1:1, y1:0.466409, x2:0.188, y2:0.482727, stop:0.823864 rgba(82, 185, 143, 255), stop:1 rgba(115, 185, 153, 255));
padding: 8px 16px;
color: white;
font-size: 20px;
height:42px;
}
QPushButton:pressed{
color: black;
background: #008bd1

}


</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>371</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="horizontalSpacing">
        <number>5</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="btn_signup">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>KAYIT OL</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLineEdit" name="txt_mail">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
         <property name="placeholderText">
          <string>info@mailadress.com</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txt_soyadi">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Soyadı</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLineEdit" name="txt_sifre">
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::PasswordEchoOnEdit</enum>
         </property>
         <property name="placeholderText">
          <string>Şifre</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLineEdit" name="txt_kullaniciAdi">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Kullanıcı Adı</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="txt_adi">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Adı</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLineEdit" name="txt_sifreTekrar">
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::PasswordEchoOnEdit</enum>
         </property>
         <property name="placeholderText">
          <string>Tekrar Şifre</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QComboBox" name="cmb_vardiya">
         <item>
          <property name="text">
           <string>Sabah Vardıyası</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Akşam Vardıyası</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gece Vardıyası</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>369</width>
        <height>142</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/resimler/icons/damla.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>369</width>
        <height>69</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #7E8BA3;
font: 44px &quot;Helvetica Neue&quot;, sans-serif;
margin: 0px 0px 16px;</string>
      </property>
      <property name="text">
       <string>KAYIT OL</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txt_adi</tabstop>
  <tabstop>txt_soyadi</tabstop>
  <tabstop>txt_kullaniciAdi</tabstop>
  <tabstop>txt_sifre</tabstop>
  <tabstop>txt_sifreTekrar</tabstop>
  <tabstop>txt_mail</tabstop>
  <tabstop>btn_signup</tabstop>
 </tabstops>
 <resources>
  <include location="../ui_pages/resoruces.qrc"/>
 </resources>
 <connections/>
</ui>
